### PR TITLE
[PSMDB-285] Build mongo-tools with new scripts

### PR DESCRIPTION
**Pull request by EvgeniyPatlan**

### CODE REVIEW

File: percona-packaging/scripts/psmdb_builder.sh [hunk 1]
```diff
@@ -689,35 +694,32 @@ build_tarball(){
...
+	fi
     done
-    fi
     #
     cd ${WORKDIR}
     #
     # Build mongo tools
     cd ${TOOLSDIR}
+    mkdir -p build_tools/src/github.com/mongodb/mongo-tools
     rm -rf vendor/pkg
```
Comment (tplavcic) [inline percona-packaging/scripts/psmdb_builder.sh]: Minor thing: from what I see this is already included in `build.sh`

File: percona-packaging/scripts/psmdb_builder.sh [hunk 2]
```diff
@@ -689,35 +694,32 @@ build_tarball(){
...
-    . ./set_gopath.sh
+    cp -r $(ls | grep -v build_tools) build_tools/src/github.com/mongodb/mongo-tools/
+    cd build_tools/src/github.com/mongodb/mongo-tools
     . ./set_tools_revision.sh
-    mkdir -p bin
-    for i in bsondump mongostat mongofiles mongoexport mongoimport mongorestore mongodump mongotop mongooplog mongoreplay; do
-    echo "Building ${i}..."
+    sed -i 's|VersionStr="$(git describe)"|VersionStr="$PSMDB_TOOLS_REVISION"|' set_goenv.sh
+    sed -i 's|Gitspec="$(git rev-parse HEAD)"|Gitspec="$PSMDB_TOOLS_COMMIT_HASH"|' set_goenv.sh
+    . ./set_goenv.sh
```
Comment (tplavcic) [inline percona-packaging/scripts/psmdb_builder.sh]: `. ./set_goenv.sh` is also already included in build.sh
I will leave it up to you if you want to remove it or not.

File: percona-packaging/scripts/psmdb_builder.sh (hunk 1 above)
Comment (tplavcic) [reply percona-packaging/scripts/psmdb_builder.sh]: Here I meant just about `rm -rf vendor/pkg`

**Comment by EvgeniyPatlan** [comment]
> I had small cleanup comment, you can decide how to proceed.
> Just a question about the debug build binaries missing - did you try that this works?

Yes it works

**Pull request merged**